src/go/pt-pg-summary/README.rst

```diff
@@ -0,0 +1,473 @@
++--------+---------------------------------+---------+----------------------------------------------------------------------------+
+| ``-U`` | ``--username=USERNAME``         |         | User for login if not current user.                                        |
++--------+---------------------------------+---------+----------------------------------------------------------------------------+
+|        | ``--disable-ssl``               | true    | Disable SSL for the connection.                                            |
++--------+---------------------------------+---------+----------------------------------------------------------------------------+
+|        | ``--verbose``                   | false   | Show verbose log.                                                          |
++--------+---------------------------------+---------+----------------------------------------------------------------------------+
+|        | ``--debug``                     | false   | Show debug information in the logs.                                        |
++--------+---------------------------------+---------+----------------------------------------------------------------------------+
+
+
+.. Currently hidden
+..       --list-encrypted-tables              Include a list of the encrypted tables in all databases
+..       --ask-pass                           Prompt for a password when connecting to PostgreSQL
+..       --config                             Config file
+..       --defaults-file                      Only read PostgreSQL options from the given file
+..       --read-samples                       Create a report from the files found in this directory
+..       --save-samples                       Save the data files used to generate the summary in this directory
+
+
+Output
+^^^^^^
+
+The output is grouped into these categories:
+
+AllDatabases
+    Selects ``datname`` from ``pg_database`` where ``datistemplate`` is false.
+
+ClusterInfo
+    Selects cluster information from ``pg_stat_activity``.
+
+ConnectedClients
+    Counts the connected clients by selecting from ``pg_stat_activity``.
+
+Connections
+    Selects ``state`` from ``pg_stat_activity`` and counts them.
+
+Counters
+    Selects various counter values from ``pg_stat_database``.
+
+DatabaseWaitEvents
+    Shows database wait events from ``pg_locks``, ``pg_stat_database``, ``pg_class``, and ``pg_stat_activity``.
+
+Databases
+    Shows the name and size of databases from ``pg_stat_database``.
+
+GlobalWaitEvents
+    Shows global wait evens from ``pg_stat_activity``.
+
+IndexCacheHitRatio
+    Shows index hit ratios from ``pg_statio_user_indexes``.
+
+PortAndDatadir
+    Shows port and data directory name from ``pg_settings``.
+
+ServerVersion
+    Shows the value of ``server_version_num``.
+
+Setting
+    Selects ``name`` and ``setting`` from ``pg_settings``.
+
+SlaveHosts10
```

Comment:
Just mention Slave Hosts.
In my opinion, wow it works internally on Postgress 9.x or 10.x shouldn't be mentioned here.